nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,51 @@
+const express = require("express");
+const app = express();
+
+// import data here
+const meals = require("./data/meals");
+const reservations = require("./data/reservations");
+const reviews = require("./data/reviews");
+
+// arrays for responses
+const mealsReviewed = JSON.parse(JSON.stringify(meals)).map((meal) => {
```

Comment:
The composition of parse and stringify (JSON.parse after JSON.stringify) gives you the identity function here so you could write meals here.

Reply:
I've used parse-stringify to make a deep copy of meals insead of referring to it. Is it unnecessary? If so, should i remove it, and then i am free to merge?